src/components/common/helpers/time.ts

```diff
@@ -35,3 +35,8 @@ export function handleUTCTime(ts: string, isRelativeTime = false) {
     }
     return timestamp;
 }
+
+export const formatDurationDiff = (startedOn: string, finishedOn: string) => {
```

Comment:
We can  use inbuilt formatters

Reply:
Yes the built in formatter was not formatting time in the way that we wanted it to be displayed, so I created my own formatter, Sohel  checked the code and told me to do so.